Azure-Samples/containerapps-albumapi-java
# Dockerize java sample app to be consistent and fix missing root (/) message

## Purpose
This app needs to be consistent with the others. We will create a buildpack branch for the non-Docker flow after.

## Does this introduce a breaking change?
```
[X] Yes
[ ] No
```
This will have some impact on some of the docs because we now have a Dockerfile. Changes to docs forthcoming.

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[X] Other... Please describe: see message above
```

## How to Test
*  Get the code
*  docker build -t albumjava .
*  docker run -it -p 8080:8080 albumjava
*  access http://localhost:8080/ to observe message



## What to Check
Verify that the following are valid
* image builds and runs after (see command above)
* / produces /albums message
